packages/syft/src/syft/service/worker/utils.py

```diff
@@ -353,9 +353,6 @@ def create_kubernetes_pool(
                 "CREATE_PRODUCER": "False",
                 "INMEMORY_WORKERS": "False",
                 "OTEL_SERVICE_NAME": f"{pool_name}",
-                "OTEL_PYTHON_LOG_CORRELATION": os.environ.get(
```

Comment:
Q: We are removing this, as currently the log co-relation is not working?

Reply:
because it configures `logging.basicConfig` but not uvicorn logger through the YAML file.